Monika After Story/game/script-topics.rpy

```diff
@@ -14736,6 +14736,44 @@ label monika_dying_same_day:
 
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_away",
+            category=["Monika"],
+            prompt="What do you do when I'm away?",
+            pool=True,
+            aff_range=(mas_aff.NORMAL, None)
+        )
+    )
+
+label monika_player_away:
+    m 1eub "I do a lot of things when you're away, [player]."
+    m 1eua "I often like to read whatever new short stories I can find online."
+    m 3eub "I've found some really interesting ones out there you can read for free!"
```

Comment:
This is begging for a few actual recommendations. Perhaps if you know 2 or 3 that Monika would be apt to recommend, add them here.